design-system/uilib/src/components/dpdhl-checkbox/dpdhl-checkbox.stories.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+	import { Meta, Template, Story } from "@storybook/addon-svelte-csf";
+	import "@sprinteins/dpdhl-uilib";
+</script>
+
+<Meta title="11_Components/Checkbox" component={null} />
+
+<Template let:args />
+
+<Story
+	name="Primary"
+	args={{
+		disabled: false,
+		indeterminate: false,
+		error: false,
+		checked: true,
+	}}
+	let:args
+>
+  	<main>
+		<dpdhl-checkbox
+			on:check={() => console.log("Checked")}
+			name="checkbox-example"
+			disabled={args.disabled}
+			checked={args.checked}
+			indeterminate={args.indeterminate}
```

Comment:
What should happen if both `checked` and `indeterminate` are `true`?

Would be a state `state: "none", "indeterminate", "checked"` better?